src/libraries/System.Private.CoreLib/src/System/ObjectDisposedException.cs

```diff
@@ -44,6 +45,14 @@ protected ObjectDisposedException(SerializationInfo info, StreamingContext conte
             _objectName = info.GetString("ObjectName");
         }
 
+        [DoesNotReturn]
+        public static void Throw(object instance) =>
```

Comment:
As the existing constructors allow for null.

```suggestion
        public static void Throw(object? instance) =>
```

Reply:
This was discussed and the conclusions was it should not be nullable:
https://github.com/dotnet/runtime/issues/51700#issuecomment-924242084